data/datasets/synthetic/anomalyPoint/amplitude/constantPointAnomalyLowDensitySet_10minute.csv

```diff
@@ -1198,7 +1198,7 @@ float,float,int,bool
 39.8000,  1.00000000,1,0
 39.8333,  1.00000000,1,0
 39.8667,  1.00000000,1,0
-39.9000,  1.00000000,1,0
+39.9000,  1.20000000,1,1
```

Comment:
seems this wasn't working correctly before, good we rerun the dataset generation.